builtin/logical/pki/path_roles.go

```diff
@@ -376,6 +389,35 @@ func (b *backend) pathRoleCreate(
 	return nil, nil
 }
 
+func parseKeyUsages(input string) int {
+	var parsedKeyUsages x509.KeyUsage
+	splitKeyUsage := strings.Split(input, ",")
+	for _, k := range splitKeyUsage {
+		switch strings.ToLower(k) {
+		case "digitalsignature":
+			parsedKeyUsages = parsedKeyUsages | x509.KeyUsageDigitalSignature
```

Comment:
Ah yeah. This used to be an `append` to an array in early code, and I just did some `sed`ing.